src/main/java/Game.java

```diff
@@ -162,4 +172,134 @@ private Position getPositionFromUser() {
         }
     }
 
+    @NotNull
+    private Player getTheWinner() {
+        for (Map.Entry<Position, Cell> entry : board) {
```

Comment:
Iterating through positions only is enough, the Board iterator could iterate through positions only